backend/tests/integration/disambiguation/test_utils.py

```diff
@@ -196,13 +196,15 @@ def test_link_signatures_to_author_missing_uuid(inspire_app):
 @freeze_time("2019-02-15")
 def test_create_new_empty_author(inspire_app):
     author = create_new_empty_author()
+    control_number = author["control_number"]
     author.pop("control_number")
     expected_data = {
         "name": {"value": "BEARD STUB"},
         "_collections": ["Authors"],
         "stub": True,
         "acquisition_source": {"method": "beard", "datetime": "2019-02-15T00:00:00"},
         "$schema": "http://localhost:5000/schemas/records/authors.json",
+        "self": {"$ref": f"http://localhost:5000/api/authors/{control_number}"},
```

Comment:
If you adds `self` here then you could alse remove line 199 `author.pop("control_number") and add control number to expected_data.